crates/sui-graphql-rpc/src/types/balance.rs

```diff
@@ -46,9 +46,11 @@ pub(crate) type Cursor = cursor::JsonCursor<BalanceCursor>;
 /// `checkpoint_viewed_at` sets the consistent upper bound for the cursor.
 #[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
 pub(crate) struct BalanceCursor {
+    #[serde(rename = "ct")]
```

Comment:
...seeing as we're milking it for all its worth?

```suggestion
    #[serde(rename = "t")]
```